vault/dendron.tutorial.conclusion.md

```diff
@@ -11,23 +11,62 @@ nav_order: 4
+If you want to learn more about [[publishing your notes online|dendron://dendron.dendron-site/dendron.topic.publish.quickstart]], you can take a look at how to publish with example guides for [[Netlify|dendron.topic.publish.cook.netlify]] or [[GitHub Pages|dendron.topic.publish.cook.github]].
 
-- [ ] Check out our [Seed Bank](https://wiki.dendron.so/notes/6ff8cbb6-e4b8-449b-a967-277b76e4ecef.html) to pull down documentation for public topics.
+![[dendron://dendron.dendron-site/asset.preview#publishing-nextjs-site-preview,1:#*]]
 
+## Explore the full potential of Dendron
 
-### Other Resources
-- Fellow dendrologist [Ian Jones](https://github.com/theianjones) made a fantastic video series about getting started with Dendron [here](https://egghead.io/courses/build-a-personal-knowledge-management-system-with-dendron-b24b)
\ No newline at end of file
+There is much more that Dendron offers. When you're ready, learn more about additional features with this _Choose Your Own Adventure_ list of learnings. Each section represents and area of a map that goes deeper into the world of Dendron.
+
+### Trail
+
+- Basics
+    - [x] Learn the basics of Dendron with the [[Getting Started tutorial|dendron.tutorial]].
+- Navigation
+    - Learn how to [[find all note references to a header|dendron://dendron.dendron-site/dendron.topic.navigation#find-all-references]].
+    - Learn how to [[find all broken, internal reference links|dendron://dendron.dendron-site/dendron.topic.navigation#findbrokenlinks]].
+    - Learn how to navigate quickly to related notes using [[Go to Commands|dendron://dendron.dendron-site/dendron.topic.navigation#go-to]].
+- Organizing
+    - Learn to expand your organization beyond hierarchies by [[using tags|dendron://dendron.dendron-site/dendron.topic.tags]].
+    - Learn how to [[tag other Users|dendron://dendron.dendron-site/dendron.topic.users]] in your notes.
+    - Learn how to [[manage and track tasks with special Task Notes|dendron://dendron.dendron-site/dendron.topic.tasks]].
+    - Learn how to quickly take notes that may note have a home quite yet, like a scratchpad, using [[Scratch Notes|dendron://dendron.dendron-site/dendron.topic.special-notes#scratch-note]].
+    - Learn how to log your activities with [[Daily Journal Notes|dendron://dendron.dendron-site/dendron.topic.special-notes#daily-journal]] and other generic [[Journal Notes|dendron://dendron.dendron-site/dendron.topic.special-notes#journal-note]].
+    - Learn the _how_ and the _why_ behind [[Hierarchies in Dendron|dendron://dendron.dendron-site/dendron.topic.hierarchies]].
+    - Learn how to automatically make use of [[Templates|dendron.topic.templates]] in new notes, in specific hierarchies, with [[Schemas|dendron://dendron.dendron-site/dendron.topic.schema]].
+- Sharing
+    - Learn how to backup your vaults online, privately or accessible to the public, with [[git and GitHub|dendron://dendron.dendron-site/dendron.topic.git]].
+
+### Forest
```

Comment:
Here we go a level deeper and go over more advanced topics. Wander into these topics to learn how to customize Dendron for exactly your workflow.